### PR TITLE
fix bugs in findMethodInvocations and improve findMethodInvocations method

### PR DESCRIPTION
**Pull request by AndyWu2015**
You can review the code and make comments , if this pull request is merged , rerun all Junit test , some of them will fail , and failed tests' author must check their example code and change the assert number .

### CODE REVIEW

**Comment by AndyWu2015** [comment]
Following is my console output:
Guess Right : arg number is right but type is not fully matched . Line :73 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :60 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :73 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :60 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :73 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :60 File:StorageAdapterCreateUpdateMethodsTest.java
Guess Right : arg number is right but type is not fully matched . Line :362 File:LiferayPortlet.java
Guess Right : arg number is right but type is not fully matched . Line :375 File:LiferayPortlet.java
Guess Right : arg number is right but type is not fully matched . Line :389 File:LiferayPortlet.java
Guess Right : arg number is right but type is not fully matched . Line :375 File:LiferayPortlet.java
Guess Right : arg number is right but type is not fully matched . Line :389 File:LiferayPortlet.java

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: blade.migrate.liferay70/src/blade/migrate/core/JavaFileChecker.java [hunk 1]
```diff
@@ -288,34 +290,50 @@ public boolean visit(MethodInvocation node) {
...
+									 }else{
+										 //type unmatched
+										 possibleMatch = false;
+										 break;
+									 }
+								}else{
+									//TODO
+									//can't resolve the type but  args number matched . make a warning ?
+									possibleMatch = false;
+									System.out.println("Guess Right : arg number is right but type is not fully matched . Line :"+_ast.getLineNumber(expression.getStartPosition()) +" File:"+_file.getName());
```
Comment (gamerson) [inline blade.migrate.liferay70/src/blade/migrate/core/JavaFileChecker.java]: Rebase onto master, now I've added a new parameter to SearchResult called "fullMatch" where you can send a true/false if it is a full or possible match.  So in this case you would create a search result where fullMatch  was false.

**Comment by gamerson** [comment]
Please rebase and set the fullMatch param and resend, thanks.

**Pull request closed without merging**